test/evergreen.yml

```diff
@@ -115,6 +115,17 @@ functions:
         rm -rf "wiredtiger"
         rm -rf "wiredtiger.tgz"
 
+  "test checkpoint for filetype":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/build_posix"
+      script: |
+        set -o errexit
+        set -o verbose
+        for fileType in m r c; do
+          ./test/checkpoint/t -t $fileType -n 1000000 -k 5000000 -C cache_size=100MB 2>&1
+        done
+
```

Comment:
I think you don't need to extract this as a function as it is only used once.

Reply:
I see why you did this. I prefer it like this
./test/checkpoint/t ${checkout_args| some default value} 2>&1
and call this three times in the job

Reply:
we can set the working dir to wiredtiger/build_posix/test/checkpoint

Reply:
Fixed. Calling the function with different arguments.